examples/redMethod/main.libsonnet

```diff
@@ -3,7 +3,7 @@ local red = import 'redDashboard/main.libsonnet';
 (
   red.new('Faro')
   + red.withPrometheusQueries('(ops|dev)-cortex', 'job=~".*/faro-.*"')
-  //  + red.withTestData()
+  //+ red.withTestData()
```

Comment:
I know this isn't what this PR is about, but I would really appreciate a comment about this comment. What is it there for. What does it offer me were I to uncomment it?

Reply:
Might be good to put this example in the docs, excellent point.